src/app/routes/index.js

```diff
@@ -1,15 +1,13 @@
 import Article from '../containers/Article';
 import getInitialData from './getInitialData';
 
+const serviceRegex = 'news|persian';
```

Comment:
If this is being extracted, it could be re-factored into an implementation that allows for simple additions of service. i.e. an array of services that is joined into a string with a `|` separator. It would be easier to maintain an array of strings than a regex string.

Reply:
Will do.